mathcomp/Makefile.test-suite.coq.local

```diff
@@ -0,0 +1,22 @@
+OUTPUT_TESTS=test_suite/output.v
+
+OUTPUT_ARTIFACTS=$(OUTPUT_TESTS:%.v=%.v.out.new)
+
+COQ_VERSION=$(shell $(COQC) -print-version | cut -d ' ' -f 1 | cut -d '.' -f 1-2)
+
+# Given a file f we compare its compilation output f.out.new with
+# f.out.COQ_VERSION (or f.out if f.out.COQ_VERSION does not exist)
+post-all:: $(OUTPUT_ARTIFACTS)
+	@for f in $(OUTPUT_TESTS); do\
+	  if [ -e $$f.out.$(COQ_VERSION) ]; then REFERENCE=$$f.out.$(COQ_VERSION);\
+	  else REFERENCE=$$f.out; fi;\
+	  diff -u $$REFERENCE $$f.out.new > /dev/null;\
+	  if [ $$? -ne 0 ]; \
+	    then echo "Output of file $$f differs!";\
+		diff -u $$REFERENCE $$f.out.new;\
+		exit 1;\
+	  fi;\
+	done
+
+$(OUTPUT_ARTIFACTS): %.v.out.new: %.v all/all.vo
```

Comment:
it seems [from the log](https://gitlab.com/math-comp/math-comp/-/jobs/729111262) that mathcomp is needlessly recompiled twice.

I guess you should just remove the `all/all.vo` prerequisite, so that the `test_suite/output.v` compilation will rely on the existing build of mathcomp (installed by the CI job using opam)

```suggestion
$(OUTPUT_ARTIFACTS): %.v.out.new: %.v
```